### ISSUE TITLE
Different encoding of value

### ISSUE THREAD

**Issue by klauspost** [pquerna/ffjson#81]
I get a difference in encoding on a single value:

![bug_81](https://cloud.githubusercontent.com/assets/5663952/6106487/1d4f11fc-b062-11e4-8d95-3971101f304f.png)

```json
ffjson:   "Q": "\"ċʇP\u003eɹ薷$ȢÝȼ@ȏL\"",
golang    "Q": "\"ċʇP\\u003eɹ薷$ȢÝȼ@ȏL\"",
```

I will submit a test that shows this, and update this bug.

**Comment by klauspost** [comment]
I got a simpler reproducer:

```Go
func TestForceStringTaggedEscape(t *testing.T) {
	testType(t, &TstringTagged{X: `"\""`}, &XstringTagged{X: `"\""`})
}
```

**Comment by klauspost** [comment]
Actually ffjson seems to be right... I will keep you updated...

**Comment by klauspost** [comment]
ok, "string" seem to indicate double-escaping of string values. See http://play.golang.org/p/4RGH6YAjNF

**Comment by klauspost** [comment]
Should be fixed by including PR #83
